qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1 Send CV.feature

```diff
@@ -0,0 +1,185 @@
+Feature: 3.1 Send CV.feature
+
+Background:
+Given user opened the page with the form to send CV
+
+Scenario Outline: 3.1_01 Different cases entering data in fields "Name", "Email", attach CV, mark checkbox
+When user enters name <Name>
+And user enters email <Email>
+And user <attach_notattach> CV <CV>
+And user <mark_notmark> checkbox <checkbox>
+And user pushes the "Send" button
+Then result <result> should be <error_noerror>
+Examples:
+|Name   |Email  |CV        |checkbox|result  |
+|valid  |valid  |attach    |mark    |no error|
+|valid  |valid  |attach    |not mark|error   |
+|valid  |valid  |not attach|mark    |error   |
+|valid  |invalid|attach    |mark    |error   |
+|invalid|valid  |attach    |mark    |error   |
+|valid  |invalid|not attach|not mark|error   |
+|invalid|valid  |not attach|not mark|error   |
+|invalid|invalid|attach    |not mark|error   |
+|invalid|invalid|not attach|mark    |error   |
+|invalid|invalid|not attach|not mark|error   |
+|valid  |valid  |not attach|not mark|error   |
+|valid  |invalid|attach    |not mark|error   |
+|valid  |invalid|not attach|mark    |error   |
+|invalid|invalid|attach    |mark    |error   |
+|invalid|valid  |not attach|mark    |error   |
+|invalid|valid  |attach    |not mark|error   |
+
+Scenario: 3.1_02 Invalid lendth data in the field "Name"
+When user enters "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras tempor rhoncus convallis. Mauris tristique ex et sapien egestas,- non malesuada nisl porttitor" in the "Name" input
+And user pushes the "Send" button
+Then options inputs should have "The maximum number of characters is 70" validation error
```

Comment:
There I suggest to write from user side like before:
Then user sees the validation error "The maximum number of characters is 70

Reply:
Thanks for your comment! 
But I slightly disagree with this it: according to this article ( it's official information about Cucumber and Gherkin syntax)  https://cucumber.io/docs/gherkin/reference/   in the step "When" we should describe actions and in this case we can write how a person interacting with the system. But in the next step "Then" we should describe something that comes out of the system (report, user interface, message), so I mean, concrete results (text of validation error in my case). That's why I think according to this article, it's unnecessary to add information from the user's point of view.


![Screenshot_1](https://user-images.githubusercontent.com/94701044/143896696-aec43f61-e426-42b9-b8db-10d4de394530.png)
![Screenshot_2](https://user-images.githubusercontent.com/94701044/143896702-da3ce6c2-774a-4e17-bd97-4e4e1d177dc5.png)

Reply:
Okey, you are right :)